File: Dia52/NumerosBinarios.py
# Funciones y números binarios en Python

def binario(numero):
  binario = ''
  while numero > 1:
    residuo = numero % 2
    binario = str(round(residuo)) + binario
    numero = (numero - residuo) / 2
  binario = str(round(numero)) + binario
  return binario

print(binario(14)) # 1110
print(binario(38)) # 100110
print(binario(127)) # 111111

File: Dia52/NumerosBinarios.js
// Funciones y números binarios en JavaScript

function binario(numero) {
  let binario = ''
  while (numero > 1) {
    let residuo = numero % 2
    binario = `${residuo}${binario}`
    numero = (numero - residuo) / 2
  }
  binario = `${numero}${binario}`
  return binario
}

console.log(binario(14)) // 1110
console.log(binario(38)) // 100110
console.log(binario(127)) // 111111